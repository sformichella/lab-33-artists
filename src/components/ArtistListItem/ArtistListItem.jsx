import React from 'react';
import PropTypes from 'prop-types';

export default function ArtistListItem({ name }) {
  return <>{name}</>;
}

ArtistListItem.propTypes = {
  name: PropTypes.string.isRequired
};
