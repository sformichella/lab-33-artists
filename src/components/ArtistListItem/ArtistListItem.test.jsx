import React from 'react';
import { render, cleanup } from '@testing-library/react';
import ArtistListItem from './ArtistListItem';

describe('ArtistListItem component', () => {
  afterEach(() => cleanup());
  it('renders ArtistListItem', () => {
    const { asFragment } = render(<ArtistListItem
      name="Michael Jackson"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
