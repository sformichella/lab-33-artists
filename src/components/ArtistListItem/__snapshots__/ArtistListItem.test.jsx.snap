// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistListItem component renders ArtistListItem 1`] = `
<DocumentFragment>
  Michael Jackson
  <div />
</DocumentFragment>
`;
